modBoot/resource/config.xml

```diff
@@ -3,7 +3,7 @@
 	<mode>aion</mode>
 	<id>[NODE-ID-PLACEHOLDER]</id>
 	<api>
-		<rpc active="true" ip="127.0.0.1" port="8545">
+		<rpc active="true" ip="0.0.0.0" port="8545">

```

Comment:
config file change should not be merged in

Reply:
Will be fixed